app/src/main/java/org/mozilla/fenix/Presenter.kt

```diff
@@ -0,0 +1,34 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix
```

Comment:
Can we get rid of this file and the `SavedLoginsPresenter`? I don't see how they're used and I'd rather not add new complexity to our current arch if we can get around it